enet.pyx

```diff
@@ -798,6 +828,9 @@ cdef class Event:
                 (<Packet> self._packet)._enet_packet = self._enet_event.packet
             return self._packet
 
+from weakref import WeakValueDictionary
+cdef Host_static_instances = WeakValueDictionary()
```

Comment:
It's a C callback, so not. We need to map ENetHost instances to pyenet Host instances in order to be able to store python data structures and have them available by C _enet_host handle.